Sources/SwiftToolchain/ToolchainManagement.swift

```diff
@@ -291,4 +284,27 @@ extension FileSystem {
 
     return result.sorted()
   }
+  
+  public func fetchLocalSwiftVersion() throws -> String? {
+    guard let cwd = currentWorkingDirectory else { return nil }
+
+    let versionFile = cwd.appending(component: ".swift-version")
+
+    guard isFile(versionFile), let version = try readFileContents(versionFile)
+      .validDescription?
+      // get the first line of the file
+      .components(separatedBy: CharacterSet.newlines).first,
+      version.contains("wasm")
```

Comment:
Please make `fetchLocalSwiftVersion()` return the version as is, without any checks for the `wasm` substring:

```suggestion
      .components(separatedBy: CharacterSet.newlines).first
```